packages/nimble-components/src/anchor/tests/anchor.stories.ts

```diff
@@ -33,15 +32,15 @@ const metadata: Meta<AnchorArgs> = {
     // prettier-ignore
     render: createUserSelectedThemeStory(html`
         <style class='code-hide'>
-            * {
+            span {
                 font: var(${bodyFont.cssCustomProperty});
             }
         </style>
-        Click on the <${anchorTag}
+        <span>Click on the <${anchorTag}
```

Comment:
Can you give the span a class name to target (just in-case span's get emitted in mdx as well to avoid conflicts) and also the `code-hide-top-container` so it's not shown in the showcode view. Actually the `code-hide-top-container` class may be enough to target.

Reply:
I don't feel strong enough for any particular option. Showing the span is probably fine.